app/controllers/application_controller.rb

```diff
@@ -1,4 +1,16 @@
 # frozen_string_literal: true
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def append_info_to_payload(payload)
```

Comment:
I guess, but my point is that enhancing the payload with this info could be a good practice, so everyone at Decidim has it.